volsupervisor/loop.go

```diff
@@ -16,6 +19,33 @@ var (
 	volumeMutex = &sync.Mutex{}
 )
 
+func (dc *DaemonConfig) updateVolumes() {
+	myVolumes, err := dc.Config.ListAllVolumes()
+	fmt.Println(myVolumes)
+	if err == nil {
```

Comment:
sure.

On 22 Aug 2016, at 13:58, Bill Robinson wrote:

>> @@ -16,6 +19,33 @@ var (
>>  	volumeMutex = &sync.Mutex{}
>>  )
>>
>> +func (dc *DaemonConfig) updateVolumes() {
>> +	myVolumes, err := dc.Config.ListAllVolumes()
>> +	fmt.Println(myVolumes)
>> +	if err == nil {
>
> We generally seem to avoid else clauses elsewhere in the codebase when 
> possible.  Can we just return from a `err != nil` and then unindent 
> the rest of the function?
>
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/435/files/c7d8dfbeaadd372ad97222a731b97939c5c5d542..e1d8c9c912325bd3cc9499e2277ec812df753329#r75757379